Client/U2FExtensions.swift

```diff
@@ -164,6 +171,24 @@ class U2FExtensions: NSObject {
         observeKeyStateUpdates = false
     }
     
+    private func handleCancelButton() -> PopupViewDismissType {
```

Comment:
Since we're DRY'ing up the previous usage of this, we could potentially in-line this, but not a big deal either way)